setup-machine.sh

```diff
@@ -346,10 +346,13 @@ sudo chown -R vx-services:vx-services /var/vx/data
 sudo chmod -R u=rwX /var/vx/data
 sudo chmod -R go-rwX /var/vx/data
 
-# config readable & executable by all vx users, writable by admin.
+# Config is writable by the vx-admin user and readable/executable by all vx-* users.
+# /vx/config/REBOOT_TO_VENDOR_MENU is special-cased to be writable by all vx-* users.
+sudo echo 0 > /var/vx/config/REBOOT_TO_VENDOR_MENU
 sudo chown -R vx-admin:vx-group /var/vx/config
 sudo chmod -R u=rwX /var/vx/config
-sudo chmod -R g=rX /var/vx/config
+sudo chmod -R g=rX  /var/vx/config
+sudo chmod    g=rwX /var/vx/config/REBOOT_TO_VENDOR_MENU
```

Comment:
One subtlety here is that I’m granting the whole group write permissions on this file, so vx-admin, vx-services, and vx-ui, when really only the first 2 need that access. I figured not a deal breaker but happy to find a way to make the permission more targeted to just those two users, if need be

Reply:
There's a defense-in-depth argument here to not let vx-ui do this, but I'm okay with this being a followup ticket.